packages/components/src/progress-bar/styles.ts

```diff
@@ -70,7 +69,7 @@ export const Indicator = styled.div< {
 					width: `${ INDETERMINATE_TRACK_WIDTH }%`,
 			  } )
 			: css( {
-					width: `${ value }%`,
+					width: 'var(--indicator-width)',
```

Comment:
Can be removed if we end up setting `width` directly through the `style` prop of the component.

```suggestion
```

Reply:
If we do, it could yeah.